src/webhook/handlers/tutorial.js

```diff
@@ -0,0 +1,387 @@
+import { t } from 'ttag';
+import {
+  ellipsis,
+  createPostbackAction,
+  createReplyMessages,
+  FLEX_MESSAGE_ALT_TEXT,
+} from './utils';
+
+/**
+ * Fixed inputs that indicate which reply should `tutorial` function return
+ */
+// From rich menu
+export const RICH_MENU_TRIGGER = `📖 ${t`tutorial`}`;
+// From flex message button
+const SIMULATE_FORWARDING_MESSAGE = t`Simulates forwarding a message`;
+// From quick reply, Note: it should be less than 20 charactors
+const PROVIDE_PERMISSION_SETUP = `💡 ${t`Cool, I got it!`}`;
+const EXPLAN_CHATBOT_FLOW_AND_PROVIDE_PERMISSION_SETUP = `🤔 ${t`What happened?`}`;
+const PROVIDE_PERMISSION_SETUP_WITH_EXPLANATION = `❓ ${t`Why permission`}`;
```

Comment:
Since `Why setup permissions` is more than 20 characters, I suggest we just use `Why`